cmd/private_network_create.go

```diff
@@ -87,8 +137,8 @@ func (c *privateNetworkCreateCmd) cmdRun(_ *cobra.Command, _ []string) error {
 	if !globalstate.Quiet {
 		return (&privateNetworkShowCmd{
 			cliCommandSettings: c.cliCommandSettings,
-			PrivateNetwork:     *privateNetwork.ID,
-			Zone:               c.Zone,
+			PrivateNetwork:     op.Reference.ID.String(),
+			Zone:               v3.ZoneName(c.Zone),
```

Comment:
No need of this cast if you updated the zine definition in ` privateNetworkCreateCmd`